js/base/Exchange.js

```diff
@@ -2566,6 +2566,15 @@ module.exports = class Exchange {
         return result;
     }
 
+    isTriggerOrder (params) {
+        const paramValue = this.safeValue2 (params, 'trigger', 'stop');
+        const isTrigger = paramValue === true;
```

Comment:
this check is redundant, if paramValue === true means that paramValue itself is already the true boolean

Reply:
Just thought it might avoid inputting price mistakenly by user i.e. `trigger: 123.455` , but you are right, no need to guard against it.